packages/embark/templates/demo/app/components/blockchain.js

```diff
@@ -1,5 +1,5 @@
 import EmbarkJS from 'Embark/EmbarkJS';
-import SimpleStorage from 'Embark/contracts/SimpleStorage';
+import SimpleStorage from '../../embarkArtifacts/contracts/SimpleStorage';
```

Comment:
As mentioned on Gitter, I believe at the moment importing from `Embark/contracts/...` should still work. EmbarkJS code still gets generated. Looping in @jrainville to confirm.

Either way though, I think the demos should run on embark-artifacts, so this change is good.

Reply:
Yeah, it should work, but I have had issues with the test_app with using `Embark/contracts`, so either a problem with linking or something.
Anyway, I agree that we should use the embarkArtifacts